docs/setup/getting_started.soy

```diff
@@ -153,30 +432,24 @@ project_config(
 )" > res/com/example/activity/BUCK
```

Comment:
since you're already here, please kill the project_config entries :) They were needed for the now-deprecated Intellij project generator.